src/components/Assets/AssetMap.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Map from "pigeon-maps";
+import Marker from "pigeon-marker";
+import { FIELDS } from "./fieldConfig";
+
+const AssetMap = props => {
+  const fields = FIELDS.ASSETS_MAP;
+  const { name, location } = fields;
+  const data = props.data;
+  const locationData = data[location];
+  const coordinates = locationData
+    ? [
+        parseFloat(locationData.latitude, 5),
+        parseFloat(locationData.longitude, 5),
+      ]
+    : null;
+  return (
+    <div className="container">
+      <h3>{data[name]}</h3>
+      {locationData && (
+        <Map center={coordinates} zoom={16} height={400}>
```

Comment:
I'm wondering how the map looks on a tablet. Sure its fine, but maybe worth a test.

Reply:
Definitely worth looking at - looks good to me in BrowserStack. I really like Pigeon Maps for this feature so I'm glad you suggested it for this.